pkg/kubeadm/node/csr.go

```diff
@@ -34,7 +35,15 @@ import (
 )
 
 func getNodeName() string {
-	return "TODO"
+	nodename, err := os.Hostname()
```

Comment:
What do you mean? I don't think there's a helper like this in the kubelet code...